geomet_data_registry/layer/base.py

```diff
@@ -189,6 +197,64 @@ def update_count(self, item, r, item_dict):
         else:
             self.new_key_store = True
 
+    def check_layer_dependencies(self, layers_list, str_mr, str_fh):
+        """
+        Checks if all layer dependencies are available in the tileindex
+        for a given model run and forecast hour.
+        :param layers_list: `list` of layer dependencies
+        :param str_mr: `str` of model run
+        :param str_fh: `str` of forecast hour
+        :returns: `list` of GeoJSON objects for retrieved dependencies
+        """
+        dependencies = [self.tileindex.get('{}-{}-{}'.format(layer, str_mr, str_fh)) for layer in layers_list]  # noqa
+
+        if None not in dependencies:
+            return dependencies
+
+    def configure_layer_with_dependencies(self,
+                                          dependencies,
+                                          image_dimension,
+                                          bands_order):
+        """
+        Create VRT and joined weather_variable string for layers
+        that consist of multiple weather variables.
+        :param dependencies: `list` of GeoJSON objects from tileindex
+        :param image_dimension: `dict` with x and y keys
+        :param bands_order: `list` of variables order in VRT
+        :returns: `list` with VRT and combined weather variables as string
+        """
+        filepaths = [self.filepath] + [dependency['properties']
+                                       ['filepath'] for dependency
+                                       in dependencies]
+        vrt = VRTDataset(
+            filepaths,
+            raster_x_size=image_dimension['x'],
+            raster_y_size=image_dimension['y'],
+            bands_order=bands_order).build()
+
+        weather_variables = '{},{}'.format(
+            self.wx_variable, ",".join(
+                [dependency['properties']['weather_variable']
+                 for dependency in dependencies]))
+
+        return vrt, weather_variables
+
+    def check_dependencies_default_mr(self, mr_datetime, dependencies):
+        """
+        For each dependency, verify that a default model run is available in
+        the store and the value to the passed model run time.
+        :param dependencies: `list` of dependencies to check
+        :param mr_datetime: `datetime` to compare dependencies to
+        :returns: True if each dependency's default model run is equal to
```

Comment:
Should this be `bool` of whether each dependency's default model run is equal to the `mr_datetime` parameter ?

Reply:
Yes, modified the docstring to make this more clear.